app/helpers/container_summary_helper.rb

```diff
@@ -82,8 +82,7 @@ def textual_guest_applications
 
   def textual_container_image_registry
     object = @record.container_image_registry
-
-    if object.nil?
+    if object.nil? && @record.class == ContainerImage
```

Comment:
@moolitayer this should be at least `kind_of?`... although even that is not optimal.
Best thing is to handle that with `respond_to?`. cc @zeari